SharelaneAutomationTest/SharelaneAutomationTest/Tests/Login.cs

```diff
@@ -0,0 +1,45 @@
+using NUnit.Framework;
+using OpenQA.Selenium;
+using OpenQA.Selenium.Chrome;
+using SharelaneAutomationTest.Pages;
+using System;
+
+namespace SharelaneAutomationTest
+{
+    internal class Login : BaseTest
+    {
+        [Test]
+        public void Test1() 
+        {
+            string email = "amit_wang@955.33.sharelane.com";
+            string password = "1111";
+            string welcomeText = "Hello Amit";
```

Comment:
for this data better to  use models and  it's nice to have faker for random data